app/program.js

```diff
@@ -1,11 +1,11 @@
-let OSinfo = require('../modules/OSinfo');
+const OSinfo = require('../modules/OSinfo');
 
 process.stdin.setEncoding('utf-8');
-process.stdin.on('readable', function() {
-    var input = process.stdin.read();
-    if(input !== null) {
-        var instruction = input.trim();
-        switch(instruction) {
+process.stdin.on('readable', function () {
+    const input = process.stdin.read();
+    if (input !== null) {
```

Comment:
instead of nesting switch in if you can write

```js
if (!input) {
  return;
}
```

Reply:
done